src/components/Modals/FiatRamps/config.ts

```diff
@@ -42,7 +42,7 @@ export interface SupportedFiatRampConfig {
   order: number
   isActive: (featureFlags: FeatureFlags) => boolean
   getBuyAndSellList: () => Promise<[AssetId[], AssetId[]]>
-  onSubmit: (action: FiatRampAction, asset: AssetId, address: string) => void
+  onSubmit: (action: FiatRampAction, asset: AssetId, address: string, language: string) => void
```

Comment:
I believe this should be `LanguageTypeEnum` and carried over from the `Preferences` slice type
Can we update it over this PR as well as in https://github.com/shapeshift/web/blob/5e1acb0656138d12a95c33d3bef50a6c287ad9ba/src/state/slices/preferencesSlice/preferencesSlice.ts#L35